Sources/Hummingbird/Middleware/FileMiddleware.swift

```diff
@@ -44,24 +44,28 @@ public protocol FileMiddlewareFileAttributes {
 public struct FileMiddleware<Context: RequestContext, Provider: FileProvider>: RouterMiddleware where Provider.FileAttributes: FileMiddlewareFileAttributes {
     let cacheControl: CacheControl
     let searchForIndexHtml: Bool
+    let urlBasePath: String?
     let fileProvider: Provider
 
     /// Create FileMiddleware
     /// - Parameters:
     ///   - rootFolder: Root folder to look for files
+    ///   - urlBasePath: Prefix to remove from request URL
     ///   - cacheControl: What cache control headers to include in response
     ///   - searchForIndexHtml: Should we look for index.html in folders
     ///   - threadPool: ThreadPool used by file loading
     ///   - logger: Logger used to output file information
     public init(
         _ rootFolder: String = "public",
+        urlBasePath: String? = nil,
```

Comment:
One more question: Can we change `urlBasePath` to use `RouterPath` instead? That would clarify the API a lot more for me.

Reply:
> One more question: Can we change `urlBasePath` to use `RouterPath` instead? That would clarify the API a lot more for me.

Making it a RouterPath makes sense

Reply:
Actually it doesn't make sense. Using a router path implies we support wildcards, parameters when loading files